### DISCUSSION TITLE
2 different instance environment sharing same database

### DISCUSSION THREAD

**Discussion by Hardik12c** [kagkarlsson/db-scheduler#551] [Q&A]
Just giving an example. Like i have 2 instances of db-scheduler 
one is called beta and one is blue 

now i have created a generic scheduler which can schedule an object and pick it and based on the class implementation, it processes it, this code is deployed in both instances.
Now i have added some new code with a new implementation of that object in beta instance, I am scheduling it through beta but sometimes it's getting consumed on blue server and As the new implemenation is not there in blue instance. it's not getting processed. 

I want to get some solution like if anything is getting scheduled through beta then it should consume on beta.

**Comment by kagkarlsson** [comment]
You might want to use two different tables. All scheduler-instances using a table must have all task implementations that occurs in that table
